### ISSUE TITLE
Update dark mode

### ISSUE THREAD

**Issue by kalicki1** [privacy-tech-lab/optmeowt-browser-extension#132]
While I was reorganizing our dependencies, I found that our dark mode package was the only dependency I had to leave a modified copy of in our source code. If it is possible, maybe we could find a way to avoid this kind of code use and rely on a stock npm package or something related. 

In addition, we should also clean up how we render dark mode on our extension. Given we have a few new elements it'd be nice to make everything look as clean in dark mode as it does in our regular light mode.

**Comment by SebastianZimmeck** [comment]
Good point! Let's do it.

**Comment by kalicki1** [comment]
I just finalized the dark mode changes and the UI updates for this issue. Here is how everything currently looks at the moment: 

<img width="948" alt="Screen Shot 2021-01-19 at 9 56 43 PM" src="https://user-images.githubusercontent.com/60238705/105121040-696e4180-5aa1-11eb-82a3-ec6475e13274.png">

<img width="949" alt="Screen Shot 2021-01-19 at 9 59 27 PM" src="https://user-images.githubusercontent.com/60238705/105121154-9e7a9400-5aa1-11eb-835f-5393b3238bee.png">

<img width="338" alt="Screen Shot 2021-01-19 at 9 57 24 PM" src="https://user-images.githubusercontent.com/60238705/105121051-6c693200-5aa1-11eb-822f-867f740fa764.png">

<img width="350" alt="Screen Shot 2021-01-19 at 10 00 37 PM" src="https://user-images.githubusercontent.com/60238705/105121273-ccf86f00-5aa1-11eb-90fc-4a372cbc0e44.png">

One small note about the general UI changes is that I took out the dividing lines in the popup both in dark mode and in light mode since it gives an overall cleaner look. I will open up a pull request for these changes and we can close this issue if we think this design looks good. Let me know if you all have any thoughts @SebastianZimmeck @dknopf !
